### PR TITLE
DYN-3818-GuidedTours-MenuEntries

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

I added the "Get Started" submenu and the InteractiveGuide MenuItem, so we can show the Get Started popups.
Also in the DynamoView.xaml.cs I added the code needed for creating most of the popups for the GetStarted tour (the content of the popup is read from the Resources file, that's why I added several entries in the resources).

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@QilongTang

### CODE REVIEW

File: src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs [hunk 1]
```diff
@@ -2322,6 +2324,143 @@ private void Window_PreviewMouseLeftButtonUp(object sender, MouseButtonEventArgs
...
+        }
+
+        /// <summary>
+        /// This method probably will be modified or deleted in the future when the GuideManager and Guide class are created
+        /// For now will be used just for testing/demo purposes since the popups will be created probably in the Guide class.
+        /// </summary>
+        private void ShowGetStartedGuidedTour()
+        {
+
+            Step.TotalSteps = 6;
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs:2339]: Can we define this 6 as const somewhere?

**Comment by QilongTang** [comment]
@RobertGlobant20 Positions of all popups are currently fixed, are we going to work on APIs to get the dynamically?

**Comment by RobertGlobant20** [comment]
> @RobertGlobant20 Positions of all popups are currently fixed, are we going to work on APIs to get the dynamically?

@QilongTang Could you give me more details about what you mean by fixed?
If you are talking about the creation of the six popups in the DynamoView.xaml.cs, yes, in future, we will need to implement the functionality for Guide class and GuidesManager class, then I think the popups creation will be done in the Guide class, my idea was to use a xml where the "guides creator" will define all the guides and steps and then read all the xaml and the popups should be created automatically with all the settings coming from the xml but I still need to work on that design in order to decide if all the information will be deserialized or not.

File: src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs (hunk 1 above)
Comment (RobertGlobant20) [reply src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs:2339]: @QilongTang This method (private void ShowGetStartedGuidedTour()) was added just for testing purposes, so Aabishkar will be able to validate all the PopupWindow UI implemented (when the SurveyWindow UI is finished also will be added in this method), then I'm not sure if is worth to create a constant for the value 6.
Thanks

Comment (QilongTang) [reply src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs:2339]: I see, then dont worry about it

**Review by QilongTang — APPROVED** [review]
LGTM

**Pull request merged**